app/views/staff/rooms/update.js.erb

```diff
@@ -1,19 +1,25 @@
-$('#room-edit-<%= room.id %>').modal('hide');
+<% if @room.save %>
```

Comment:
I noticed the call to #save below for track modal update and left a comment there. Same applies here.